js/tooltip.js

```diff
@@ -385,13 +385,54 @@
     return delta
   }
 
+  Tooltip.prototype.didModuleReceiveTypeStringOrFunction = function (module, received) {
+    if (!/string|function/.test(received)) {
+      throw new Error(module + ' received type ' + received + '. Was expecting type string or a function.')
+    }
+
+    return true
+  }
+
+  Tooltip.prototype.didFunctionReturnStringOrJQuery = function (module, returned, returnedType) {
```

Comment:
This doesn't handle normal DOM nodes being returned which would also be added by #14552.